integration_tests/src/main/python/spark_init_internal.py

```diff
@@ -55,6 +55,20 @@ def _spark__init():
     _s.sparkContext.setLogLevel("WARN")
     return _s
 
+
+def _create_derby_dir(sb, driver_opts, wid):
```

Comment:
`_create_*_dir` feels like all it should do is to create that dir, not setup/change configuration.  We either need a better name or split this up.